developer-docs-site/docs/tutorials/run-a-local-testnet.md

```diff
@@ -175,20 +173,23 @@ Completed generating configuration:
 Aptos is running, press ctrl-c to exit
 ```
 
-This output contains information required for starting the Aptos CLI tool:
+Use the [Aptos CLI tool](https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos/README.md) to interact with your local testnet. The above output contains information you will use for starting the [Aptos CLI tool](https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos/README.md):
```

Comment:
This is accurate? I mean, the CLI tool is used to interact with the local testnet, I think?